base-template/base-template.md

```diff
@@ -2,20 +2,120 @@
+<br>This would be content that is:
+<ol>
+<li>Repetitive/similar in terms of:
+<ol>
+<li>Structure</li>
+<li>Formatting, styles</li>
+<li>Procedures </li>
+<li>Text such as descriptions, disclaimers, snippets etc.</li>
+  </ol></li>
+that can be grouped together, written/presented in the same manner, and reused.<br>
+<br>
+<li>Does not have existing templates to accommodate above.</li><br>
+<li>Existing templates require modification to accommodate the above.</li></ol><br>
+
+<li>Identify a prototype, i.e., a  suitable sample document that you are going to base your template on. This document will have the most variety of the content that you will be covering, in your template.</li><br>
+
+<li>Identify the type of documentation you need to write the template for. This would broadly be one, or a combination of the following types of documentation:
+<ul>
+<li>How-to guide</li>
+<li>Tutorial</li>
+<li>Reference</li>
+  <li>Explanation</li></ul><br>
 
-**Keywords:** Optionally add comma-separated keywords.
+<li>Identify the repetitive content in your documents. This could be for instance,
+<ul>
+<li>Introduction - a generic intro that gets repeated in all docs. Could be that of the organization, department, documentation etc.</li>
+<li>Disclaimers, policies and other legal snippets that are applicable throughout.</li>
+<li>Structure and formatting such as headers, footers, font sizes, font types etc.</li>
+  <li>Processes and procedures. </li></ul><br>
 
-## Content 1
+<li>Identify your target audience, so as to be able to give clear instructions on creating a new template,  from the perspective of your reader.</li><br>  
 
-### Subsection 1
+<li>Create the template that you are going to need, giving out step-by-step instructions, in clear, simple language. <br>
 
-## Content 2
+Although the specific template you are creating may vary, ensure that you have covered the basics of a good template. <br>
 
-## Reference
+Some of the essential elements of your template would be:
+<ul>
+<li>Title - include a title. The title should be reflective of what your template is for. </li>
+<li>Overview - a brief description of what your template can be used for.</li>
+<li>Prerequisites / before you start </li>
+<li>Procedures</li>
+<li>Images / screenshots</li>
+<li>Code snippets - include a sample snippet with expected formatting and styles.</li>
+<li>Optionally, repetitive content like, organization introduction, disclaimer, glossary etc.</li>
+<li>Page structure - header, title, page numbers, table of content etc.</li>
+<li>Styles</li>
+<li>Directory structure</li>
+<li>Naming conventions for directories, files, documents etc.</li>
+</ul>
+</li>
+<li>Use consistent language and tone throughout your docs.</li><br>
+
+<li>Once you have created the template that you require, you can optionally add it to the existing archive of Good Docs Project Templates, to be used by the wider community.
+</ol><br>
 
 ## Examples
```

Comment:
I'd be inclined to suggest that we shouldn't create example templates.
Reason being is that the best examples we have are our existing templates (which hopefully will be updated to follow our base template).